docs/Example.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { ThemeProvider } from 'styled-components'
-import Box from '../src/Box'
+import Box from '../src/styled-components/Box'
```

Comment:
Yeah, I literally threw this together on my couch last night, so not sure that Box is needed here. The intent is to use this lib in grid-styled, etc